### DISCUSSION TITLE
clarity is only for angular, or it can be used with vue3

### DISCUSSION THREAD

**Discussion by ericxin1982** [vmware-clarity/core#312] [Q&A] [answered]
Hi All,

It is framework based, or it is free to use with any framework?

Thanks
Eric Xin

**Comment by kevinbuhmann** [comment, marked as answer]
Clarity Core (`@cds/core`) is a web component library, so it can be used with any framework (or without a framework). However, I'll warn you that these component are not currently actively supported.

**Reply by ericxin1982** [reply]
Ok, thanks.
